### DISCUSSION TITLE
System change

### DISCUSSION THREAD

**Discussion by bolbolfolfol** [LSPosed/LSPatch#65] [Ideas]
Hi
i have rooted device Aosp10
can i just copy the JAR file to framework and install the APK after reboot ?
will it work ?
and does it support "Xposed Edge" and "GravityBox" ?
and thank you so much

**Comment by david082321** [comment]
Why don't you use LSPosed?

**Reply by bolbolfolfol** [reply]
Hi
i already use it but it require 2 modules ,riru+lsposed (or zygisk)
i want to try this because only 2 files
